### DISCUSSION TITLE
when having several buffers open, list them and highlight the one under use

### DISCUSSION THREAD

**Discussion by jerabaul29** [helix-editor/helix#6983] [Q&A] [answered]
I have several buffers open in my helix session. I can cycle between them with ```gn``` and ```gp```, so I know "they are here". But I only can see the name of the currently active buffer in the bottom status bar. Any way to get the status bar to list the different buffers I have open, with the current one a bit highlighted, a bit like what vim does with tabs?

**Comment by the-mikedavis** [comment, marked as answer]
This is supported with a bufferline feature. See the "Bufferline" section of the 22.12 release notes: https://helix-editor.com/news/release-22-12-highlights/#bufferline

**Reply by jerabaul29** [reply]
I thought this was all I needed, but actually, I looked in more details, and it looks like only the name (actually, full path) of the current file can be displayed, not a list of the different files, like in the screenshot:

![image](https://user-images.githubusercontent.com/8382834/236700544-51022a05-9be6-4f40-8b2a-b652027dddfd.png)

Any way to get something similar, i.e. here the ".bashrc" highlighted (because it is in use), while the ".bash_aliases" is also shown as it is also an opened buffer (even if not rendered for now)? If this is not possible to get at the moment, should I re open the issue / feature request? If this is doable, can you point to how to get it? (somehow I cannot find out how to get this working this way from looking into the manual :) ).

**Reply by kirawi** [reply]
From what I understood, that looks like how Helix behaves: https://helix-editor.com/news/release-22-12-highlights/#bufferline

**Reply by jerabaul29** [reply]
Aaahh, sorry, I was confused between the bufferline and the line at the bottom. Thanks for your patience, sorry again.
